starknet_py/contract.py

```diff
@@ -216,11 +215,8 @@ async def invoke(
         )
         response = await self._client.send_transaction(transaction)
 
-        if response.code != StarkErrorCode.TRANSACTION_RECEIVED.name:
-            raise Exception("Failed to send transaction. Response: {response}.")
-
```

Comment:
This is a quite big removal in terms of functionality. I think it'd be good to include a warning/note about it in the docs. cc @Arcticae

Reply:
yep, agreed